content/en/blog/2024/sig-arch-code-spotlight.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: blog
+title: "Spotlight on SIG Architecture: Code Organization"
+slug: sig-architecture-code-spotlight-2024
+date: 2024-04-04
+author: "Frederico Muñoz (SAS Institute)"
+---
+
+_This is the third interview of a SIG Architecture Spotlight series that will cover the different
+subprojects. We will cover the [SIG Architecture: Code
+Organization](https://github.com/kubernetes/community/blob/master/sig-architecture/README.md#code-organization-1)_
+
+In this SIG Architecture spotlight we talked with [Madhav
```

Comment:
(nit)
```suggestion
In this SIG Architecture spotlight I talked with [Madhav
```
There were two people talking, one was Madhav, so you'd use first person singular.

Reply:
Yes, old habits that are not adequate here.